docs/docs/overview/supported-tests.mdx

```diff
@@ -0,0 +1,81 @@
+import DocCard from "../../src/theme/DocCard";
+import clsx from "clsx";
+
+# Supported Test Types
+
+Testkube supports running a long list of testing tools and frameworks. You can find the list of supported test types below.
+
+In case your test tool/framework is not currently support, Testkube supports creation of custom Test Types, known as Testkube Container executors. It enables you to run your own container images for executing tests. You can check the guide [here](../test-types/container-executor.md) for more details.
+
+An alternative method to creating your own Test Type is by creating [Custom Testkube Executor](../test-types/executor-custom.md) and configure it to run any type of tests that you want. These custom executors can be added to your Testkube installation and/or contributed to our repo. Although we recommend the use of Testkube Container executors moving forward.
```

Comment:
An alternative method to creating your own Test Type is by creating [Custom Testkube Executor](../test-types/executor-custom.md) and configuring it to run any type of tests that you want. These custom executors can be added to your Testkube installation and/or contributed to our repo. We recommend the use of Testkube Container executors moving forward.